content/uiux/journey-map/journey-map.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: 'Journey Maps in UX'
+Description: ''
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'UI'
+- 'UX'
+- 'Design'
```

Comment:
can you please explain the `tag.md` folder changes that you've done? Alot of those changes are not used here.

Reply:
I only added (ui ux and design) I figured those are appropraite tags for the doc I am working on. no?